build.sbt

```diff
@@ -105,6 +105,22 @@ lazy val `semantic-ui-react-kitchensink` = project
     )
   )
 
+lazy val `semantic-ui-react-redux` = project
```

Comment:
can we deemphasize semantic-ui since it has its own demo and just call it react-redux?